README.md

```diff
@@ -23,11 +23,24 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+This library needs these to be in order to build and run:
+
+- Node 16 or Node 18
```

Comment:
I think we should mention Node20 and can even remove Node 16

Reply:
Done 👍